### ISSUE TITLE
set row_major or column_major when initializing Namelist from dict

### ISSUE THREAD

**Issue by gitporst** [marshallward/f90nml#74]
Hello,

I am glad to see that version 1.0.2 is out. I am doing the following:
```
d = {'sec': {'array': [[1, 2], [3,4]]}}
nml = f90nml.Namelist(d, default_start_index=1)
f = io.StringIO()
nml.write(f)
f.seek(0)
f.read()
```
and get
`'&sec\n    array(1:2, 1) = 1, 2\n    array(1:2, 2) = 3, 4\n/\n'`.

I would like to get
`'&sec\n    array(1, 1:2) = 1, 2\n    array(2, 1:2) = 3, 4\n/\n'`, which I think would be more intuitive for my use case. 

I saw that the `Parser` class has a `_row_major` attribute, maybe something like this would be good for the `Namelist` class or the `write` method, also?! How can I get my desired output? Thanks a lot.

**Comment by marshallward** [comment]
Just checking, you want this
```
d = {'sec': {'array': [[1, 2], [3,4]]}}
```
to produce this namelist?
```
&sec\n array(1:2, 1) = 1, 3\n array(1:2, 2) = 2, 4\n/\n'
```
Yes, that feature was added to `Parser` but not `Namelist` (another case of me not thinking much about initialization with `dict`).

I think it might even make more sense to move that code into `Namelist`, but I'll see what can be done.  (Not for a while, unfortunately, due to some other commitments, but maybe next week or so)

BTW glad to see someone using this.  I was not sure it would be useful when I implemented it.

**Comment by gitporst** [comment]
Yes, that would be it, what I want. Thank you very much and no rush.
